src/Tools/dotnet-user-secrets/test/InitCommandTest.cs

```diff
@@ -88,6 +89,18 @@ public void DoesNotGenerateIdForProjectWithSecretId()
             Assert.Equal(SecretId, idResolver.Resolve(null, null));
         }
 
+        [Fact]
+        public void DoesNotAddXmlDeclarationToProject()
+        {
+            var projectDir = _fixture.CreateProject(null);
+            var projectFile = Path.Combine(projectDir, "TestProject.csproj");
+
+            new InitCommand(null, null).Execute(MakeCommandContext(), projectDir);
+
+            var projectDocument = XDocument.Load(projectFile);
+            Assert.False(projectDocument.Declaration is XDeclaration);
```

Comment:
What's the actual value here? Is it `null`? Perhaps an `Assert.Null` would be more appropriate?

Reply:
Thanks for the suggestion! Changed it.